### ISSUE TITLE
Issue with training 3D U-Net using ZeroCostDL4Mic: validation set

### ISSUE THREAD

**Issue by jbogomolovas2** [HenriquesLab/ZeroCostDL4Mic#266]
First of all, I want to express my gratitude towards ZeroCostDL4Mic. It is an amazing tool that I use for almost every image analysis task in my work. However, I am currently facing an issue with training a 3D U-Net to recognize bright cell clones in light-sheet microscopy images.

To train the model, I have provided 22 labeled 128x128x128 pixel images. I have varied the validation set (20-50%), batch size (1-5), and patch size (64-128x64-128x8-32), but the validation loss function does not converge. As seen in the attached image, the validation set behaves way worse than the training set.
![Unknown-11](https://github.com/HenriquesLab/ZeroCostDL4Mic/assets/105531636/9f0a0c07-a3d8-46f2-ad9c-3e3d48140eb0)
I also came across a similar issue reported on the image.sc forum:
https://forum.image.sc/t/u-net-3d-zerocostdl4mic-training/79457

To investigate the issue further, I ran a sanity check on the dataset by repeating the same 128x128x128 labeled image 10 times and using a 50% validation fraction. However, even in this case, the validation set behaved differently than the training set. :
![Unknown-13](https://github.com/HenriquesLab/ZeroCostDL4Mic/assets/105531636/3dc94c9a-8a38-473a-bb24-360fb6a63c40)

Shouldn't the validation and training sets behave identically in this test case? This suggests that there might be a bug in the script.

I would appreciate any help or suggestions to resolve this issue. Thank you in advance!

**Comment by jing00011** [comment]
I have similar graph before and adding elastic deformation + choosing default parameters helped with the issue.

**Comment by jbogomolovas2** [comment]
@jing00011 Thank you for suggestion. So activate elastic deformation and keep the rest default?
